lib/tesla_api/client.rb

```diff
@@ -91,5 +91,13 @@ def vehicles
     def vehicle(id)
       Vehicle.new(self, email, id, self.get("/vehicles/#{id}")['response'])
     end
+
+    def api_uri
+      base_uri + '/api/1/'
+    end
+
+    def base_uri
+      BASE_URI
```

Comment:
We don't need the constant. Having both is confusing. Just move the value down to this method.